website/package.json

```diff
@@ -18,9 +18,9 @@
     "@hashicorp/react-button": "^6.0.1",
     "@hashicorp/react-call-to-action": "^4.0.0",
     "@hashicorp/react-code-block": "^4.1.5",
-    "@hashicorp/react-consent-manager": "^7.1.0",
+    "@hashicorp/react-consent-manager": "^7.1.2",
```

Comment:
Bumps to pull in hashicorp/react-components#514, to fix a weird `events`-related build error.